docs/digger/admin/monitoring.asciidoc

```diff
@@ -0,0 +1,12 @@
+== Monitor Digger
+Nagios is provisioned by default in the Digger Ansible playbook. The Nagios
+instance will perform a number of checks.
+
+* Ensure the macOS machine is running and accessible
+* Ensure the Jenkins instance is running
+* Ensure the Android SDK pod is running.
```

Comment:
I don't think the Android SDK pod needs to be running all the time. It's really only required for installing the SDK to PV. After that I think it's ok to stop and remove it.